Remotion/ObjectBinding/Core/BusinessObjectStringFormatterService.cs

```diff
@@ -189,10 +189,10 @@ private string GetStringValue (IBusinessObject businessObject, object value, IBu
       if (property is IBusinessObjectStringProperty)
         return GetStringValueForStringProperty (value);
 
-      return (value != null) ? value.ToString() : string.Empty;
+      return (value != null) ? value.ToString()! : string.Empty;
```

Comment:
should we add a guard here, too, same as with the other ToString?

Or should we do a null-coalesce to string.Empty in both cases?

Reply:
I think string.Empty fallback would be the better choice.